src/main/java/net/earthcomputer/clientcommands/command/FindBlockCommand.java

```diff
@@ -73,6 +78,13 @@ protected boolean canScanChunkSection(Entity cameraEntity, SectionPos pos) {
             return hasBlockState(pos, predicate::canEverMatch) && super.canScanChunkSection(cameraEntity, pos);
         }
 
+        @Override
+        public boolean condition() {
+            boolean condition = hasChunksRemaining() || (keepSearching && closestBlock == null);
```

Comment:
The reason for this override is to stop searching when a block has been found and there are no more loaded chunks to search. I think this code could be cleaner if you set `keepSearching` to `false` after you find the block. This has the same effect and is logically marginally clearer IMO. It also removes the need for this override.